kebab/src/entities.ts

```diff
@@ -1,4 +1,9 @@
 import { ConsumerOptions } from "mediasoup-client/lib/types";
+import {
+  GetScheduledRoomsResponse,
+  GetTopPublicRoomsResponse,
+  JoinRoomAndGetInfoResponse,
+} from "./responses";
```

Comment:
ideally, entities.ts isn't supposed to import responses.ts. you messed something up

Reply:
How should I get the types then? They aren't imported anywhere else in the file and I need them to define the Wrapper typings.

Reply:
use `ReturnType<typeof wrap>` (it's predefined in `wrapper.ts`)